doc/esp32_overview.gv

```diff
@@ -0,0 +1,396 @@
+    <TD ROWSPAN="5">13</TD>
+      <TD BGCOLOR="lightgray" PORT="XTAL32_1">XTAL32_1</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="TOUCH8">TOUCH8</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO8">RTCIO8</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC1_5">ADC1_5</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO33">GPIO33</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="5">IO25</TD>
+    <TD ROWSPAN="5">14</TD>
+      <TD BGCOLOR="lightgray" PORT="DAC1">DAC1</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO6">RTCIO6</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC2_8">ADC2_8</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="EMAC_RXD0">EMAC_RXD0</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO25">GPIO25</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="5">IO26</TD>
+    <TD ROWSPAN="5">15</TD>
+      <TD BGCOLOR="lightgray" PORT="DAC2">DAC2</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO7">RTCIO7</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC2_9">ADC2_9</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="EMAC_RXD1">EMAC_RXD1</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO25">GPIO25</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="5">IO27</TD>
+    <TD ROWSPAN="5">16</TD>
+      <TD BGCOLOR="lightgray" PORT="TOUCH7">TOUCH7</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO17">RTCIO17</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC2_7">ADC2_7</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="EMAC_RXDV">EMAC_RXDV</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO27">GPIO27</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="6">IO14</TD>
+    <TD ROWSPAN="6">17</TD>
+      <TD BGCOLOR="lightgray" PORT="TMS">TMS</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="TOUCH6">TOUCH6</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO16">RTCIO16</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC2_6">ADC2_6</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="EMAC_TXD2">EMAC_TXD2</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO14">GPIO14</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="6">IO12</TD>
+    <TD ROWSPAN="6">18</TD>
+      <TD BGCOLOR="lightgray" PORT="TDI">TDI</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="TOUCH5">TOUCH5</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="RTCIO15">RTCIO15</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="ADC2_5">ADC2_5</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="EMAC_TXD3">EMAC_TXD3</TD></TR><TR>
+      <TD BGCOLOR="lightgray" PORT="GPIO12">GPIO12</TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="1">GND</TD>
+    <TD ROWSPAN="1" BORDER="0"></TD>
+    <TD ROWSPAN="1" BORDER="0"></TD>
+  </TR>
+  <TR><TD BORDER="0"> </TD></TR>
+  <TR>
+    <TD ROWSPAN="9">IO13</TD>
+    <TD ROWSPAN="9">20</TD>
+      <TD BGCOLOR="lightgray" PORT="HS2CLK">HS2CLK</TD></TR><TR>
```

Comment:
While ```U0_TXD``` and ```ADC1_0``` are highlighted in green ```HS2CLK``` isn't. Is this intended?

Reply:
It isn't, fixed it